apps/meteor/client/startup/e2e.ts

```diff
@@ -146,11 +147,33 @@ Meteor.startup(() => {
 				return message;
 			}
 
+			const mentionsEnabled = settings.get<boolean>('E2E_Enabled_Mentions');
+
+			if (mentionsEnabled) {
+				const me = (message.u?._id && Users.findOne(message.u?._id, { fields: { username: 1 } })?.username) || '';
```

Comment:
Why not use `Meteor.user()` to get the username?

Reply:
I thought that we are trying to stop using Meteor methods as much as possible... in this case is this necessary?

Reply:
With that logic, we are also trying to stop using mini-mongo collections.

And we will replace mini-mongo collections much before we start worrying about Meteor.

Reply:
make sense! Thanks